.gitignore

```diff
@@ -115,19 +115,8 @@ public/stylesheets/jmaki-standard-right-sidebar.css
 # tmp/
 tmp/*
 
-# vendor/bundle
```

Comment:
Just vendor/bundle :)

Though now I'm remembering more....I added this because my entry in `~/.gitignore` didn't work when the app was in vmdb, so shouldn't be a problem for me anymore. But I would keep it in at least in this PR as it's not related to your change. Thanks!